src/ebonite/ext/flask/build_templates/flask-site-nginx.conf

```diff
@@ -1,4 +1,6 @@
 server {
+    listen       9000;
```

Comment:
please check docs and example which maybe affected by this change

Reply:
I didn't find anything. I couldn't be sure but it looks reasonable as we always used port 9000 for all servers and it is even hardcoded in the client as default port.

Reply:
this is internal port in container, so maybe there are wrong port mappings. full search for string "80" in source would reveal that, i think

Reply:
"wrong port mappings" - wrong for what? I want you to remember that our servers doesn't encapsulate port number. Therefore it's important to employ the same port for all servers. Otherwise both examples and `DockerContainer.__init__` which specifies `{9000: 9000}` as default mapping will work or break depending on a server being in charge. Flask and asyncio use 9000, client - 9000, nginx Flask - 80 (which is nginx default). That's the reason for change here.

I performed search and found that Jupyter example used nginx Flask and relied on port 80. Fixed it. Now we could change server implementation in this tutorial without touching ports.

Reply:
i meant they still use 80 instead of 9000. no all is ok